lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,19 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(locales_path = GettextSetup.locales_path)
+      mergepot = GettextSetup.config['project_name'] + '.pot'
+      cmd = "msgcat #{locales_path}/*.pot > #{locales_path}/#{mergepot}"
```

Comment:
Maybe add a `--force` or `--overwrite` argument then, which would allow overwriting an existing file? The behavior of overwriting a project-specific file with data from other projects' files seems counter-intuitive (and probably doesn't make much sense as a default, I'd say).